app/assets/javascripts/app/app_router.coffee

```diff
@@ -0,0 +1,21 @@
+Router = Backbone.Router.extend {
+  routes:
+    ':name'   : 'show'
+
+  show: (name) ->
+    @showPage(name)
+
+  showPage: (name) ->
```

Comment:
In this case, we dont really need both `show` and `showPage` functions since one just proxies to the other.   Your handler could just be `':name' : 'showPage'` instead.